pages/common/idnits.md

```diff
@@ -0,0 +1,25 @@
+# idnits
+
+> Check internet-drafts for submission nits.
+> Looks for violations of Section 2.1 and 2.2 of the requirements listed on <https://www.ietf.org/id-info/checklist>.
+> More information: <https://tools.ietf.org/tools/idnits/>.
+
+- Check a file for nits:
+
+`idnits {{path/to/file.txt}}`
+
+- Count the nits without displaying them:
```

Comment:
```suggestion
- Count nits without displaying them:
```
I think that sounds a bit better here.

Reply:
Ah, that's a lot nicer! I was having trouble with that line, but you really unlocked it 😄